src/VisualStudio/LiveShare/Test/HandlerShimsTests.cs

```diff
@@ -22,7 +22,7 @@ public void TestLiveShareRequestHandlersExported()
 
             // Verify there are exactly the number of liveshare request handlers as expected.
             // Liveshare shims will verify there is a matching roslyn request handler when they are created.
-            Assert.Equal(21, handlers.Length);
+            Assert.Equal(19, handlers.Length);
```

Comment:
Is this test really necessary? I assume we have tests too that each handler actually _works_, right? :smile:

Reply:
yep these will be deleted in a followup PR as above.